examples/getting_started/custom_problem_type_and_dynamics.py

```diff
@@ -49,10 +49,12 @@ def prepare_ocp(biorbd_model_path, ode_solver=OdeSolver.RK):
     objective_functions = {"type": Objective.Lagrange.MINIMIZE_TORQUE, "weight": 100}
 
     # Dynamics
-    problem_type = {"type": ProblemType.CUSTOM, "configure": custom_configure, "dynamic": custom_dynamic}
-
+    if problem_type_custom:
+        problem_type = {"type": ProblemType.CUSTOM, "configure": custom_configure, "dynamic": custom_dynamic}
+    else:
+        problem_type = {"type": ProblemType.TORQUE_DRIVEN, "dynamic": custom_dynamic}
     # problem_type = {"type": ProblemType.TORQUE_DRIVEN, "dynamic": custom_dynamic}  # only custom dynamic
-    # problem_type = {"type": ProblemType.CUSTOM, "function": custom_torque_driven, "dynamic": custom_dynamic}  # custom problem_type and dynamic
+    # problem_type = {"type": ProblemType.CUSTOM, "configure": custom_torque_driven, "dynamic": custom_dynamic}  # custom problem_type and dynamic
```

Comment:
These two line seems useless?

Reply:
They show how to have other configuration than all custom, we can remove the duplicate

Reply:
Aren't they just copy of the above lines?